prime-router/docs/changelog-vs-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# Changelog vs Release Notes
+
+The [release notes](./release-notes.md) are a high-level summary of changes to ReportStream over time. 
+You can also see a more detailed list of changes in the
+[list of releases](https://github.com/CDCgov/prime-reportstream/releases).
+
+The categories used to separate the changes in each release are generated by a Github Action called
+[Release Changelog Builder](https://github.com/mikepenz/release-changelog-builder-action).
+It will look through the PRs associated with a particular release and group them based on their labels. 
+The configuration used to define each category is in
+[a json file](/.github/changelog_config.json), but broadly the labels associated with each category are:
+
+
+**Category: Backend**
+
+* api
```

Comment:
What is the difference between backend and router?

Reply:
I think these tags were found in existing PRs and we tried to make them as inclusive as possible. If you think it's a good idea we can narrow down the list of labels we advertise here but keep the ones in the configuration file to make sure we catch as many issues as possible.

Reply:
Yeah, it would be nice to limit the labels if we can.  Future action.